README.md

```diff
@@ -138,6 +128,32 @@ loader.load(
 );
 ```
 
+### Use with WebGPURenderer
+
+Starting from v3, we provide a WebGPURenderer compatibility.
+For now, in order to use with WebGPURenderer, you need to specify a WebGPU compatible MToon loader plugin to the option `mtoonMaterialPlugin` of `VRMLoaderPlugin`.
+
+```js
+import { VRMLoaderPlugin, MToonNodeMaterialLoaderPlugin } from '@pixiv/three-vrm';
+
+// ...
+
+// Register a VRMLoaderPlugin
+loader.register((parser) => {
+
+  // create a WebGPU compatible MToon loader plugin
+  const mtoonMaterialPlugin = new MToonNodeMaterialLoaderPlugin(parser);
+
+  return new VRMLoaderPlugin(parser, {
+
+    // Specify the MToon loader plugin to use in the VRMLoaderPlugin instance
+    mtoonMaterialPlugin,
```

Comment:
Question: Can this be detected inside three-vrm? Should we mention `renderer.isWebGPURenderer`?

Reply:
I think `VRMLoaderPlugin` doesn't have an access to `renderer`.

Reply:
> Can this be detected inside three-vrm
So this is no.

> Should we mention

I mean if the renderer may be created before and exist somewhere in current execution context, so user can figure out what to use via `isWebGPURenderer`.

Reply:
Renderer and VRMLoaderPlugin should be loosely coupled I think. What does the code you expected look like?

Reply:
Something like
```ts
const mtoonMaterialPlugin = renderer.isWebGPURenderer === true 
  ? new MToonNodeMaterialLoaderPlugin(parser)
  : undefined
```

I know this is totally optional but may provide some useful info to users.

Reply:
I see I see.

https://github.com/pixiv/three-vrm/pull/1420/files#r1642356443 👈 Does this look good?